docs/shape_batch_size_and_layout.md

```diff
@@ -29,3 +29,17 @@ it ignores the batch_size value.
 on [Shape Inference Document](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_ShapeInference.html).
 In case the model can't be reshaped, it will remain in the original parameters and all requests with incompatible input format
 will get an error. The model server will also report such problem in the logs.
+
+# Changing model input/output layout
+OpenVINO models which process image data are usually generated via the model optimizer with NCHW layout. Image transformation libraries like OpenCV or Pillow use NHWC layout. This makes it required to transpose the data in the client application before it can be sent to OVMS. Custom node example implementations also internall use NCHW format to perform image transformations. It could increase the latency with extra data transposition operations. Layout parameter reduces the latency by changing the model in runtime to accept NHWC layout instead of NCHW. That way the whole processing cycle is more effective by avoiding unnecessary data transpositions. That is especially beneficial for models with high resolution images, where data transposition could be more expensive in processing.<br><br>
```

Comment:
not usually
custom nodes use HWC
It could increase the latency with extra data transposition operations> Transposition operations increase the overall processing latency.